src/app/starredentries/starredentries.component.html

```diff
@@ -101,44 +101,46 @@ <h4>
             </a>
             <app-starring class="pull-right" [workflow]="workflow" (starGazersChange)="starGazersChange()"></app-starring>
           </div>
-          <mat-card-header>
-            <div *ngIf="workflow?.starredUsers.length !== 0" fxLayout="row" fxLayoutAlign="end center"></div>
-            <mat-card-subtitle class="truncate-text-1">{{ workflow?.description }}</mat-card-subtitle>
-          </mat-card-header>
-          <div>
+          <div class="ml-4 mr-2">
+            <div class="truncate-text-3 mb-3">
+              <app-markdown-wrapper
+                [data]="workflow?.description"
+                [baseUrl]="workflow?.providerUrl | baseUrl: workflow?.defaultVersion"
```

Comment:
I see this pattern exists already, but @kathy-t what would it mean to have `workflow.defaultVersion` as the baseUrl? I understand the `providerUrl` -- what assets would ever get resolved against the defaultVersion?

Reply:
The workflow `defaultVersion` is an argument to the `baseUrl` pipe, used to modify certain source code control site URLs to point at a particular branch/tag. 
https://github.com/dockstore/dockstore-ui2/blob/develop/src/app/shared/entry/base-url.pipe.ts

Reply:
^^ what Steve said

It's really only used if the workflow description is from a README that contains relative links